### ISSUE TITLE
Occasional `slim -testSLiM` failure

### ISSUE THREAD

**Issue by grahamgower** [MesserLab/SLiM#269]
`slim -testSLiM` usually succeeds, but sometimes fails with:
```
slim -testSLiM
[1806] initialize() { initializeSLiMOptions(keepPedigrees=T); initializeMutationRate(1e-7); initializeMutationType('m1', 0.5, 'f', 0.0); initializeGenomicElementType('g1', m1, 1.0); initializeGenomicElement(g1, 0, 99999); initializeRecombinationRate(1e-8); } 1 { sim.addSubpop('p1', 10); } 5 { if (all(p1.individuals[0].relatedness(p1.individuals[1:9]) < 1.0)) stop(); } : FAILURE : no raise during SLiM execution.

FAILURE count: 1
SUCCESS count: 99284
```

**Comment by grahamgower** [comment]
I did manage to reproduce this just now:
```
$ i=0; while slim -testSLiM; do i=$((i+1)); done

...

[1806] initialize() { initializeSLiMOptions(keepPedigrees=T); initializeMutationRate(1e-7); initializeMutationType('m1', 0.5, 'f', 0.0); initializeGenomicElementType('g1', m1, 1.0); initializeGenomicElement(g1, 0, 99999); initializeRecombinationRate(1e-8); } 1 { sim.addSubpop('p1', 10); } 5 { if (all(p1.individuals[0].relatedness(p1.individuals[1:9]) < 1.0)) stop(); } : FAILURE : no raise during SLiM execution.

FAILURE count: 1
SUCCESS count: 99284

$ echo $i
133
```

**Comment by bhaller** [comment]
The test is incorrect after the fixes made to `relatedness()` in SLiM 3.7, I think.

**Comment by bhaller** [comment]
Sorry this took so long to fix, things are busy over here.  :->  The test was just bad; it used to be correct for the old (buggy) definition of relatedness(), but for the new (hopefully correct) definition it was sometimes violated because the test itself was no longer correct.  I just commented out the offending tests, and I think/hope all is now well.  :->  relatedness() is tested more comprehensively in a different way, outside of the unit tests run inside SLiM.
